contracts/astroport/oracle/src/error.rs

```diff
@@ -19,6 +19,12 @@ pub enum ContractError {
     #[error("Asset infos are not set")]
     AssetInfosNotSet {},
 
+    #[error("Asset infos are set")]
+    AssetInfosSet {},
+
+    #[error("Oracle is outdated, need to call Update")]
+    OracleIsOutdated {},
```

Comment:
I find this one a bit confusing… Like, when I see "outdated" I think that update was too long ago while here it means that the update has never happened. However, it's not a big issue I suppose, because you see "Outdated", you update, and it works. So I don't know if it's need to be fixed

Reply:
okay, you guys have convinced me. will make the errors better

Reply:
done